test/functional/cmdLineTests/cmdLineTest_J9tests/src/TestStringBufferAndBuilderGrowth.java

```diff
@@ -20,13 +20,25 @@
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
  *******************************************************************************/
 
+import com.sun.management.OperatingSystemMXBean;
+import java.lang.management.ManagementFactory;
+
 /**
  * @file TestStringBufferAndBuilderGrowth.java
  * @brief Grows a StringBuffer and StringBuilder to Integer.MAX_VALUE
  */
 public class TestStringBufferAndBuilderGrowth {
 
 public static void main(String[] args) {
+	OperatingSystemMXBean opBean = (OperatingSystemMXBean)ManagementFactory.getOperatingSystemMXBean();
+	long physicalMemory = opBean.getTotalPhysicalMemorySize();
+	System.out.println("Machine has physical memory " + physicalMemory + " bytes or " + (physicalMemory >> 20) + " MB or " + (physicalMemory >> 30) + " GB");
+	long tenGig = 10L << 30;
```

Comment:
I wonder whether 8GB might be a more reasonable requirement, but I suppose we can try this and experiment later.

Reply:
I know AIX doesn't work with 7616 MB, trying a 8000 MB limit.

Reply:
I think 8000MB is a more reasonable boundary.